skoolkit/snapinfo.py

```diff
@@ -404,6 +405,8 @@ def main(args):
     group = parser.add_argument_group('Options')
     group.add_argument('-b', '--basic', action='store_true',
                        help='List the BASIC program')
+    group.add_argument('-v', '--vars', action='store_true',
+                       help='List variables')
```

Comment:
Can you move these two lines below -p/--peek: options should be defined in alphabetical order.

Also use --variables instead of --vars, in case I want to use --vars later for setting VARS!